tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -674,355 +605,637 @@ describe('IssueList', function () {
+            id: '2',
+            slug: 'bar',
+            isMember: true,
+            firstEvent: false,
+          }),
+          TestStubs.Project({
+            id: '3',
+            slug: 'baz',
+            isMember: true,
+            firstEvent: false,
+          }),
+        ];
+        MockApiClient.addMockResponse({
+          url: '/organizations/org-slug/sent-first-event/',
+          query: {
+            project: [1, 2],
+          },
+          body: {sentFirstEvent: false},
+        });
+        MockApiClient.addMockResponse({
+          url: '/organizations/org-slug/projects/',
+          body: projects,
+        });
+        MockApiClient.addMockResponse({
+          url: '/projects/org-slug/foo/issues/',
+          body: [],
+        });
+
+        createWrapper({
+          selection: {
+            projects: [1, 2],
+            environments: [],
+            datetime: {period: '14d'},
+          },
+          organization: TestStubs.Organization({
+            projects,
+          }),
+        });
 
-      pushArgs = {
-        pathname: '/organizations/org-slug/issues/',
-        query: {
-          cursor: '1443575000:0:1',
-          page: 1,
-          environment: [],
-          project: [],
-          query: 'is:unresolved',
-          statsPeriod: '14d',
-        },
+        await waitFor(() => {
+          expect(wrapper.queryByTestId('loading-indicator')).toBe(null);
+        });
+
+        await waitFor(() =>
+          expect(wrapper.getByTestId('awaiting-events')).toBeInTheDocument()
+        );
+      });
+
+      it('does not display when any selected projects have first event', async function () {
+        const projects = [
+          TestStubs.Project({
+            id: '1',
+            slug: 'foo',
+            isMember: true,
+            firstEvent: false,
+          }),
+          TestStubs.Project({
+            id: '2',
+            slug: 'bar',
+            isMember: true,
+            firstEvent: true,
+          }),
+          TestStubs.Project({
+            id: '3',
+            slug: 'baz',
+            isMember: true,
+            firstEvent: true,
+          }),
+        ];
+        MockApiClient.addMockResponse({
+          url: '/organizations/org-slug/sent-first-event/',
+          query: {
+            project: [1, 2],
+          },
+          body: {sentFirstEvent: true},
+        });
+        MockApiClient.addMockResponse({
+          url: '/organizations/org-slug/projects/',
+          body: projects,
+        });
+
+        createWrapper({
+          selection: {
+            projects: [1, 2],
+            environments: [],
+            datetime: {period: '14d'},
+          },
+          organization: TestStubs.Organization({
+            projects,
+          }),
+        });
+
+        await waitFor(() => {
+          expect(wrapper.queryByTestId('loading-indicator')).toBe(null);
+        });
+
+        expect(wrapper.queryByTestId('awaiting-events')).not.toBeInTheDocument();
```

Comment:
```suggestion
  expect(wrapper.queryByText('Waiting for events…')).toBe(null);
```

Reply:
What happens if we change the `Waiting for events` text to something else? this will always pass

Reply:
yes .. I was not so sure about this one too, but at this point this text is no longer in the dom